github.com/inbucket/inbucket/v3/pkg/test is failing all the time

Hi,

on my system

```
$ uname -a
Linux moto 6.5.5-1-MANJARO #1 SMP PREEMPT_DYNAMIC Sat Sep 23 12:48:15 UTC 2023 x86_64 GNU/Linux
```
with

```
$ go version
go version go1.21.2 linux/amd64
```
at commit
```
$ git log -1
commit 3d162549b18ada5a2bdf26ffd6486394b837b7c1 (HEAD -> main, origin/main, origin/HEAD)
Author: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
```

`go test ./...` is failing with
```
?   	github.com/inbucket/inbucket/v3/cmd/client	[no test files]
?   	github.com/inbucket/inbucket/v3/cmd/inbucket	[no test files]
?   	github.com/inbucket/inbucket/v3/pkg/config	[no test files]
ok  	github.com/inbucket/inbucket/v3/pkg/extension	(cached)
?   	github.com/inbucket/inbucket/v3/pkg/extension/event	[no test files]
ok  	github.com/inbucket/inbucket/v3/pkg/extension/luahost	(cached)
?   	github.com/inbucket/inbucket/v3/pkg/metric	[no test files]
ok  	github.com/inbucket/inbucket/v3/pkg/message	(cached)
ok  	github.com/inbucket/inbucket/v3/pkg/msghub	(cached)
ok  	github.com/inbucket/inbucket/v3/pkg/policy	(cached)
ok  	github.com/inbucket/inbucket/v3/pkg/rest	(cached)
?   	github.com/inbucket/inbucket/v3/pkg/rest/model	[no test files]
ok  	github.com/inbucket/inbucket/v3/pkg/rest/client	(cached)
?   	github.com/inbucket/inbucket/v3/pkg/server	[no test files]
ok  	github.com/inbucket/inbucket/v3/pkg/server/pop3	(cached)
ok  	github.com/inbucket/inbucket/v3/pkg/server/smtp	(cached)
ok  	github.com/inbucket/inbucket/v3/pkg/server/web	(cached)
ok  	github.com/inbucket/inbucket/v3/pkg/storage	(cached)
ok  	github.com/inbucket/inbucket/v3/pkg/storage/file	(cached)
ok  	github.com/inbucket/inbucket/v3/pkg/storage/mem	(cached)
ok  	github.com/inbucket/inbucket/v3/pkg/stringutil	(cached)
?   	github.com/inbucket/inbucket/v3/pkg/webui	[no test files]
6:26PM INF Web UI content mapped module=web path=ui/dist phase=startup
6:26PM ERR Failed to parse HTML template error="open ui/dist/index.html: no such file or directory" cwd=/usr/local/src/sagu/inbucket/pkg/test module=web path=ui/dist/index.html phase=startup
6:26PM INF SMTP listening on tcp4 addr=0.0.0.0:2500 module=smtp phase=startup
6:26PM INF HTTP listening on tcp4 addr=0.0.0.0:9000 module=web phase=startup
6:26PM INF Starting SMTP session module=smtp remote=127.0.0.1:58158 session=1
6:26PM DBG Line received: EHLO localhost module=smtp remote=127.0.0.1:58158 session=1
6:26PM DBG Entering state READY module=smtp remote=127.0.0.1:58158 session=1
6:26PM DBG Line received: MAIL FROM:<fromuser@inbucket.org> BODY=8BITMIME module=smtp remote=127.0.0.1:58158 session=1
6:26PM DBG Mail sender is fromuser@inbucket.org module=smtp remote=127.0.0.1:58158 session=1
6:26PM DBG Origin domain is inbucket.org module=smtp remote=127.0.0.1:58158 session=1
6:26PM DBG ESMTP params: map[BODY:8BITMIME] module=smtp remote=127.0.0.1:58158 session=1
6:26PM INF Mail from: fromuser@inbucket.org module=smtp remote=127.0.0.1:58158 session=1
6:26PM DBG Entering state MAIL module=smtp remote=127.0.0.1:58158 session=1
6:26PM DBG Line received: RCPT TO:<recipient@inbucket.org> module=smtp remote=127.0.0.1:58158 session=1
6:26PM DBG Recipient added module=smtp remote=127.0.0.1:58158 session=1 to=recipient@inbucket.org
6:26PM DBG Line received: DATA module=smtp remote=127.0.0.1:58158 session=1
6:26PM DBG Entering state DATA module=smtp remote=127.0.0.1:58158 session=1
6:26PM DBG Delivering message mailbox=recipient module=message
6:26PM INF Message size 94 bytes module=smtp remote=127.0.0.1:58158 session=1
6:26PM DBG Entering state READY module=smtp remote=127.0.0.1:58158 session=1
6:26PM DBG Line received: QUIT module=smtp remote=127.0.0.1:58158 session=1
6:26PM DBG Entering state QUIT module=smtp remote=127.0.0.1:58158 session=1
6:26PM INF Closing connection module=smtp remote=127.0.0.1:58158 session=1
6:26PM DBG Request method=GET module=web path=/api/v1/mailbox/recipient/latest proto=HTTP/1.1 remote=127.0.0.1:60586
6:26PM INF Starting SMTP session module=smtp remote=127.0.0.1:58172 session=2
6:26PM DBG Line received: EHLO localhost module=smtp remote=127.0.0.1:58172 session=2
6:26PM DBG Entering state READY module=smtp remote=127.0.0.1:58172 session=2
6:26PM DBG Line received: MAIL FROM:<fromuser@inbucket.org> BODY=8BITMIME module=smtp remote=127.0.0.1:58172 session=2
6:26PM DBG Mail sender is fromuser@inbucket.org module=smtp remote=127.0.0.1:58172 session=2
6:26PM DBG Origin domain is inbucket.org module=smtp remote=127.0.0.1:58172 session=2
6:26PM DBG ESMTP params: map[BODY:8BITMIME] module=smtp remote=127.0.0.1:58172 session=2
6:26PM INF Mail from: fromuser@inbucket.org module=smtp remote=127.0.0.1:58172 session=2
6:26PM DBG Entering state MAIL module=smtp remote=127.0.0.1:58172 session=2
6:26PM DBG Line received: RCPT TO:<recipient@inbucket.org> module=smtp remote=127.0.0.1:58172 session=2
6:26PM DBG Recipient added module=smtp remote=127.0.0.1:58172 session=2 to=recipient@inbucket.org
6:26PM DBG Line received: DATA module=smtp remote=127.0.0.1:58172 session=2
6:26PM DBG Entering state DATA module=smtp remote=127.0.0.1:58172 session=2
6:26PM DBG Delivering message mailbox=recipient module=message
6:26PM INF Message size 123 bytes module=smtp remote=127.0.0.1:58172 session=2
6:26PM DBG Entering state READY module=smtp remote=127.0.0.1:58172 session=2
6:26PM DBG Line received: QUIT module=smtp remote=127.0.0.1:58172 session=2
6:26PM DBG Entering state QUIT module=smtp remote=127.0.0.1:58172 session=2
6:26PM INF Closing connection module=smtp remote=127.0.0.1:58172 session=2
6:26PM DBG Request method=GET module=web path=/api/v1/mailbox/recipient/latest proto=HTTP/1.1 remote=127.0.0.1:60586
6:26PM INF Starting SMTP session module=smtp remote=127.0.0.1:58188 session=3
6:26PM DBG Line received: EHLO localhost module=smtp remote=127.0.0.1:58188 session=3
6:26PM DBG Entering state READY module=smtp remote=127.0.0.1:58188 session=3
6:26PM DBG Line received: MAIL FROM:<fromuser@inbucket.org> BODY=8BITMIME module=smtp remote=127.0.0.1:58188 session=3
6:26PM DBG Mail sender is fromuser@inbucket.org module=smtp remote=127.0.0.1:58188 session=3
6:26PM DBG Origin domain is inbucket.org module=smtp remote=127.0.0.1:58188 session=3
6:26PM DBG ESMTP params: map[BODY:8BITMIME] module=smtp remote=127.0.0.1:58188 session=3
6:26PM INF Mail from: fromuser@inbucket.org module=smtp remote=127.0.0.1:58188 session=3
6:26PM DBG Entering state MAIL module=smtp remote=127.0.0.1:58188 session=3
6:26PM DBG Line received: RCPT TO:<recipient@inbucket.org> module=smtp remote=127.0.0.1:58188 session=3
6:26PM DBG Recipient added module=smtp remote=127.0.0.1:58188 session=3 to=recipient@inbucket.org
6:26PM DBG Line received: DATA module=smtp remote=127.0.0.1:58188 session=3
6:26PM DBG Entering state DATA module=smtp remote=127.0.0.1:58188 session=3
6:26PM DBG Delivering message mailbox=recipient module=message
6:26PM INF Message size 228 bytes module=smtp remote=127.0.0.1:58188 session=3
6:26PM DBG Entering state READY module=smtp remote=127.0.0.1:58188 session=3
6:26PM DBG Line received: QUIT module=smtp remote=127.0.0.1:58188 session=3
6:26PM DBG Entering state QUIT module=smtp remote=127.0.0.1:58188 session=3
6:26PM INF Closing connection module=smtp remote=127.0.0.1:58188 session=3
6:26PM DBG Request method=GET module=web path=/api/v1/mailbox/recipient/latest proto=HTTP/1.1 remote=127.0.0.1:60586
6:26PM INF Starting SMTP session module=smtp remote=127.0.0.1:58196 session=4
6:26PM DBG Line received: EHLO localhost module=smtp remote=127.0.0.1:58196 session=4
6:26PM DBG Entering state READY module=smtp remote=127.0.0.1:58196 session=4
6:26PM DBG Line received: MAIL FROM:<fromuser@inbucket.org> BODY=8BITMIME module=smtp remote=127.0.0.1:58196 session=4
6:26PM DBG Mail sender is fromuser@inbucket.org module=smtp remote=127.0.0.1:58196 session=4
6:26PM DBG Origin domain is inbucket.org module=smtp remote=127.0.0.1:58196 session=4
6:26PM DBG ESMTP params: map[BODY:8BITMIME] module=smtp remote=127.0.0.1:58196 session=4
6:26PM INF Mail from: fromuser@inbucket.org module=smtp remote=127.0.0.1:58196 session=4
6:26PM DBG Entering state MAIL module=smtp remote=127.0.0.1:58196 session=4
6:26PM DBG Line received: RCPT TO:<ip4recipient@[192.168.123.123]> module=smtp remote=127.0.0.1:58196 session=4
6:26PM DBG Recipient added module=smtp remote=127.0.0.1:58196 session=4 to=ip4recipient@[192.168.123.123]
6:26PM DBG Line received: DATA module=smtp remote=127.0.0.1:58196 session=4
6:26PM DBG Entering state DATA module=smtp remote=127.0.0.1:58196 session=4
6:26PM DBG Delivering message mailbox=ip4recipient module=message
6:26PM INF Message size 67 bytes module=smtp remote=127.0.0.1:58196 session=4
6:26PM DBG Entering state READY module=smtp remote=127.0.0.1:58196 session=4
6:26PM DBG Line received: QUIT module=smtp remote=127.0.0.1:58196 session=4
6:26PM DBG Entering state QUIT module=smtp remote=127.0.0.1:58196 session=4
6:26PM INF Closing connection module=smtp remote=127.0.0.1:58196 session=4
6:26PM DBG Request method=GET module=web path=/api/v1/mailbox/ip4recipient/latest proto=HTTP/1.1 remote=127.0.0.1:60586
6:26PM INF Starting SMTP session module=smtp remote=127.0.0.1:58210 session=5
6:26PM DBG Line received: EHLO localhost module=smtp remote=127.0.0.1:58210 session=5
6:26PM DBG Entering state READY module=smtp remote=127.0.0.1:58210 session=5
6:26PM DBG Line received: MAIL FROM:<fromuser@inbucket.org> BODY=8BITMIME module=smtp remote=127.0.0.1:58210 session=5
6:26PM DBG Mail sender is fromuser@inbucket.org module=smtp remote=127.0.0.1:58210 session=5
6:26PM DBG Origin domain is inbucket.org module=smtp remote=127.0.0.1:58210 session=5
6:26PM DBG ESMTP params: map[BODY:8BITMIME] module=smtp remote=127.0.0.1:58210 session=5
6:26PM INF Mail from: fromuser@inbucket.org module=smtp remote=127.0.0.1:58210 session=5
6:26PM DBG Entering state MAIL module=smtp remote=127.0.0.1:58210 session=5
6:26PM DBG Line received: RCPT TO:<ip6recipient@[IPv6:2001:0db8:85a3:0000:0000:8a2e:0370:7334]> module=smtp remote=127.0.0.1:58210 session=5
6:26PM DBG Recipient added module=smtp remote=127.0.0.1:58210 session=5 to=ip6recipient@[IPv6:2001:0db8:85a3:0000:0000:8a2e:0370:7334]
6:26PM DBG Line received: DATA module=smtp remote=127.0.0.1:58210 session=5
6:26PM DBG Entering state DATA module=smtp remote=127.0.0.1:58210 session=5
6:26PM DBG Delivering message mailbox=ip6recipient module=message
6:26PM INF Message size 67 bytes module=smtp remote=127.0.0.1:58210 session=5
6:26PM DBG Entering state READY module=smtp remote=127.0.0.1:58210 session=5
6:26PM DBG Line received: QUIT module=smtp remote=127.0.0.1:58210 session=5
6:26PM DBG Entering state QUIT module=smtp remote=127.0.0.1:58210 session=5
6:26PM INF Closing connection module=smtp remote=127.0.0.1:58210 session=5
6:26PM DBG Request method=GET module=web path=/api/v1/mailbox/ip6recipient/latest proto=HTTP/1.1 remote=127.0.0.1:60586
--- FAIL: TestSuite (0.51s)
    --- FAIL: TestSuite/basic (0.00s)
        integration_test.go:86: Test output did not match testdata/basic.golden
            To update golden file, run: go test -update
        integration_test.go:86: diff -want +got:
            | Mailbox: recipient
            | From: <fromuser@inbucket.org>
            | To: [<recipient@inbucket.org>]
            | Subject: basic subject
            |-Size: 217
            |+Size: 218
            | 
            | BODY TEXT:
            ...
            | 
            | 
            
    --- FAIL: TestSuite/fullname (0.00s)
        integration_test.go:115: Test output did not match testdata/fullname.golden
            To update golden file, run: go test -update
        integration_test.go:115: diff -want +got:
            | Mailbox: recipient
            | From: From User <fromuser@inbucket.org>
            | To: [Rec I. Pient <recipient@inbucket.org>]
            | Subject: basic subject
            |-Size: 246
            |+Size: 247
            | 
            | BODY TEXT:
            ...
            | 
            | 
            
    --- FAIL: TestSuite/encodedHeader (0.00s)
        integration_test.go:144: Test output did not match testdata/encodedheader.golden
            To update golden file, run: go test -update
        integration_test.go:144: diff -want +got:
            | Mailbox: recipient
            | From: X-äéß Y-äéß <fromuser@inbucket.org>
            | To: [Test of ȇɲʢȯȡɪɴʛ <recipient@inbucket.org>]
            | Subject: Test of ȇɲʢȯȡɪɴʛ
            |-Size: 351
            |+Size: 352
            | 
            | BODY TEXT:
            ...
            | 
            | 
            
    --- FAIL: TestSuite/ipv4Recipient (0.00s)
        integration_test.go:173: Test output did not match testdata/no-to-ipv4.golden
            To update golden file, run: go test -update
        integration_test.go:173: diff -want +got:
            | Mailbox: ip4recipient
            | From: <fromuser@inbucket.org>
            | To: [<ip4recipient@[192.168.123.123]>]
            | Subject: basic subject
            |-Size: 198
            |+Size: 199
            | 
            | BODY TEXT:
            ...
            | 
            | 
            
    --- FAIL: TestSuite/ipv6Recipient (0.00s)
        integration_test.go:202: Test output did not match testdata/no-to-ipv6.golden
            To update golden file, run: go test -update
        integration_test.go:202: diff -want +got:
            | Mailbox: ip6recipient
            | From: <fromuser@inbucket.org>
            | To: [<ip6recipient@[IPv6:2001:0db8:85a3:0000:0000:8a2e:0370:7334]>]
            | Subject: basic subject
            |-Size: 227
            |+Size: 228
            | 
            | BODY TEXT:
            ...
            | 
            | 
            
FAIL
6:26PM DBG SMTP shutdown requested, connections will be drained module=smtp phase=shutdown
FAIL	github.com/inbucket/inbucket/v3/pkg/test	0.517s
ok  	github.com/inbucket/inbucket/v3/pkg/webui/sanitize	(cached)
FAIL
```

--- a/pkg/server/smtp/handler.go
+++ b/pkg/server/smtp/handler.go
@@ -533,7 +533,7 @@ func (s *Session) dataHandler() {
 	mailData := bytes.NewBuffer(msgBuf)
 
 	// Mail data complete.
-	tstamp := time.Now().Format(timeStampFormat)
+	tstamp := time.Now().UTC().Format(timeStampFormat)
 	for _, recip := range s.recipients {
 		if recip.ShouldStore() {
 			// Generate Received header.

